CONTRIBUTING.md

```diff
@@ -335,7 +335,7 @@ The CFEngine codebase can be usefully thought of as a few separate components:
 utilities (libutils), parsing (libpromises), evaluation (libpromises),
 actuation (mostly in cf-agent), network (libcfnet).
 
-Over the past year, the structure of the codebase has undergone some
+During 2018/2019, the structure of the codebase has undergone some
```

Comment:
@olehermanse wasn't sure how to word this but "past year" is going to be very imprecise in a little while. ;)

Reply:
Maybe `During 2018 and 2019, the structure of the codebase underwent major changes.`? Or something similar.
I think past tense might make it easier to get a good wording, although it might highlight that the changes are still undergoing?

Reply:
I think you can say something like:

```
Since around 2016 ...
```
It has been (slowly) ongoing since I started here in 2016.